Tests/SentryTests/SentryOptionsTest.m

```diff
@@ -17,28 +17,37 @@ @interface SentryOptionsTest : XCTestCase
 
 @implementation SentryOptionsTest
 
-// TODO(fetzig): not sure if this test needs an update or SentryOptions/SentryDsn needs a fix.
-//- (void)testEmptyDsn {
-//    NSError *error = nil;
-//    SentryOptions *options = [[SentryOptions alloc] initWithDict:@{} didFailWithError:&error];
-//    XCTAssertEqual(kSentryErrorInvalidDsnError, error.code);
-//    XCTAssertNil(options);
-//}
+
+- (void)testEmptyDsn {
+    NSError *error = nil;
+    SentryOptions *options = [[SentryOptions alloc] initWithDict:@{} didFailWithError:&error];
+    
+    // TODO(fetzig): not sure if this test needs an update or SentryOptions/SentryDsn needs a fix. Maybe the error should be set to kSentryErrorInvalidDsnError
```

Comment:
Doesn't look like we need to take any actions here. Maybe we can we get rid of these todos?
```suggestion
```

Reply:
Removed, thanks :)